### PR TITLE
Feature/pelagos 4343 remove checkfileexsits api and add notification

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/js/vue/FileManager.vue [hunk 1]
```diff
@@ -246,50 +250,44 @@ const initDropzone = () => {
...
-                        .then(response => {
-                            done();
-                        }).catch(error => {
-                        currentFile.accepted = false;
-                        myDropzone._errorProcessing([currentFile], error.message);
-                    });
-                })
+                    done();
+                }).catch(error => {
+                currentFile.accepted = false;
```
Comment (mickel1138) [inline assets/js/vue/FileManager.vue:268]: Spacing?

File: assets/js/vue/FileManager.vue [hunk 2]
```diff
@@ -246,50 +250,44 @@ const initDropzone = () => {
...
-            alert(error);
-        });
+
+    myDropzone.on("totaluploadprogress", function (uploadProgress, totalBytes, totalBytesSent) {
+        // Intentionally left for testing.
+        console.log(uploadProgress, totalBytes, totalBytesSent);
+        if (uploadProgress == 100) {
+
+        }
+    });
```
Comment (mickel1138) [inline assets/js/vue/FileManager.vue:280]: Remove

File: assets/js/vue/FileManager.vue [hunk 3]
```diff
@@ -246,50 +250,44 @@ const initDropzone = () => {
...
+
+    myDropzone.on("queuecomplete", function () {
+        myFileManager.instance.repaint();
     });
 }
 
+$("#ds-submit").on("active", function() {
+    myFileManager.instance.repaint();
+});
+
```
Comment (mickel1138) [inline assets/js/vue/FileManager.vue:290]: Make javascript!

**Review by mickel1138 — APPROVED** [review]
I wrote this, its good! Seriously tho.. Approved.

**Review by praneethpr — COMMENTED** [review]
Looks good! Approved

**Review by fightingtexasaggie — APPROVED** [review]
reviewed code in-group, looks ready to go

**Pull request merged**